metapool/sample_sheet.py

```diff
@@ -962,7 +1012,11 @@ def load_sample_sheet(sample_sheet_path):
     if sheet.validate_and_scrub_sample_sheet(echo_msgs=False):
         return sheet
 
-    sheet = MetatranscriptomicSampleSheet(sample_sheet_path)
+    sheet = MetatranscriptomicSampleSheetv10(sample_sheet_path)
+    if sheet.validate_and_scrub_sample_sheet(echo_msgs=False):
+        return sheet
+
+    sheet = MetatranscriptomicSampleSheetv0(sample_sheet_path)
     if sheet.validate_and_scrub_sample_sheet(echo_msgs=False):
         return sheet
 
```

Comment:
A thought for the future: I assume that more and more defined sample sheet types/versions are in our future (given that lots and lots of not-explicitly-defined types/versions are in our past :D ).  Given this, at some point we may want to rewrite this as something like:

```
# placed somewhere easy to modify, like near the top of the module or even in a config:
ordered_sheet_factories = [AmpliconSampleSheet, MetagenomicSampleSheetv100, MetagenomicSampleSheetv90, MetatranscriptomicSampleSheetv10, MetatranscriptomicSampleSheetv0, AbsQuantSampleSheetv10]

# and then modify function to something like:
for curr_factory in ordered_sheet_factories:
    sheet = curr_factory(sample_sheet_path)
    if sheet.validate_and_scrub_sample_sheet(echo_msgs=False):
         return sheet
```

Reply:
Will do! Ty for the suggestion!